newsletter/v221.1.0-January-2023.mdx

```diff
@@ -0,0 +1,152 @@
+# January 2023 StyleGuide newsletter
+
+Combined update notes, summarising changes in the style guide since the last newsletter (since the StyleGuide v219.1.0). Some of the key highlights include:
+
+- [StyleGuide Live Editor](<#-styleguide-live-editor-(beta-version)>)
+- [Newsletter in storybook](#-newsletter-in-storybook)
+- [Focus style](#-focus-style)
+- [Dialog for ads](#-dialog-for-ads)
+- [StyleGuide Metrics](#-styleguide-metrics)
+- [Components updates](#-components-updates)
+- [Engineering](#-engineering)
+
+## ⌨️ StyleGuide Live Editor (beta version)
+
+We've created a [live editor inside the storybook](https://styleguide.brainly.com/docs/?path=/story/tools-live-editor--live-editor), so you don't have to run StyleGuide on your local environment to be able to test out StyleGuide components! This tool is in beta version and is to be improved in the future.
+
+![live editor](./newsletter-assets/2023-january/live-editor.png)
+
+## 📰 Newsletter in storybook
+
+You can now access our newsletter editions in the storybook. These pages are dynamically created upon the build process, so we have one source of truth for our newsletter, that can be easily accessed in github, but at the same time exist in the storybook!
+
+![newsletter](./newsletter-assets/2023-january/newsletter.png)
+
+## 👆 Focus style
+
+We've added two types of focus - default one, called `focusStyle` and one used for inline, wrapped elements called `focusTextStyle`.
+
+Components using `focusStyle`: `Accordion`, `Avatar`, `Button`, `Dialog`, `FileHandler`, `HomeButton`, `IconAsButton`, `Label`, `Rating`, `Search`, `SkipLink`, native interactive elements besides `<a>`.
+
+<p float="left">
+  <img
+    src="./newsletter-assets/2023-january/focus-style-button.png"
+    width="200px"
+  />
+</p>
+
+Components using `focusTextStyle`: `Link`, `Checkbox`, `Radio`, `Select`, `Dropdown`, `Search`, `Textarea`, native `<a>`.
```

Comment:
Only Link, Dropdown, and <a> use `focusTextStyle`. My bad, I didn't update the description after changes 🙈